node/src/chain_spec/crab.rs

```diff
@@ -217,6 +217,11 @@ pub fn config() -> ChainSpec {
 				ethereum: Default::default(),
 				evm: Default::default(),
 				base_fee: Default::default(),
+
+				// Darwinia <> Crab
```

Comment:
Suggest updating this to `S2s`,  These three crates might also be used in other bridges, like Crab <> Crab parachain.